src/ui/ProcessManager.js

```diff
@@ -25,12 +25,28 @@ export default class ProcessManager extends React.Component {
     return pids.indexOf(this.state.selectedPid) !== -1;
   }
 
+  canDebug() {
+    if (!this.state.selectedPid) return false;
+    const pids = this.state.processData.map(p => p.pid);
+
+    // verify that select pid is in list of processes
+    //return pids.indexOf(this.state.selectedPid) !== -1;
+
+    return true;
+  }
+
   handleKillProcess() {
     const pid = this.state.selectedPid;
     if (!pid) return;
     ipcRenderer.send('process-manager:kill-process', pid);
   }
 
+  handleDebugProcess() {
+    const pid = this.state.selectedPid;
+    if (!pid) return;
+    ipcRenderer.send('process-manager:debug-process', pid);
```

Comment:
Suggestion: I'd be more confortable passing the `webContentsId` all over rather than the `pid`

Reply:
I agree and I change it to pass webContentsId